library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java

```diff
@@ -50,9 +53,35 @@ public static String getDeletedOrMentionedText(Message message) {
                 text = text.replace("@" + userName, "**" + "@" + userName + "**");
             }
         }
+        // Markdown for newline
+        text = text.replaceAll("<br/>  <br/>  \n", "\n");
         return text.replaceAll("\n", "<br/>  <br/>  \n");
     }
 
+    private static String trimTrailingNewLine(String text){
+        if (TextUtils.isEmpty(text))
+            return text;
+        String s = text.substring(text.length() - 1);
+        while (s.equals("\n")){
+            text = text.substring(0, text.length() - 1);
+            s = text.substring(text.length() - 1);
+        }
+        return text;
+    }
+
+    private static String trimStartNewLine(String text){
+        if (TextUtils.isEmpty(text))
+            return text;
+        String s = text.substring(0, 1);
+        while (s.equals("\n")){
```

Comment:
I think you can get both with a single replace

```java
str.replaceAll("/^[\r\n]+|\.|[\r\n]+$/", "");
```

This kind of stuff is really simple and useful to test; please add some tests for all cases